Sources/Defaults/Observation.swift

```diff
@@ -125,6 +125,7 @@ extension Defaults {
 		private weak var object: UserDefaults?
 		private let key: String
 		private let callback: Callback
+		private var isObserving = false
```

Comment:
How about using [observationInfo](https://developer.apple.com/documentation/objectivec/nsobject/1414009-observationinfo) to check whether the observer is registered?

Reply:
I'm uncertain if that works because the `observationInfo` is set on the object (`UserDefaults` in this case) and not on the observer. Therefore, even if the registration of this observer has not happened, another instance might be observing the `UserDefaults`, in which case `observationInfo` is non-nil.

Reply:
> another instance might be observing the UserDefaults, in which case observationInfo is non-nil.

True, thanks for the reminder!